Common/Brokerages/ZerodhaBrokerageModel.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Orders.Slippage;
+using QuantConnect.Securities;
+using QuantConnect.Util;
+
+namespace QuantConnect.Brokerages
+{
+    public class ZerodhaBrokerageModel : DefaultBrokerageModel
+    {
+        public ZerodhaBrokerageModel(AccountType accountType = AccountType.Margin) : base(accountType)
+        {
+        }
+
+        public override AccountType AccountType => base.AccountType;
+
+        public override decimal RequiredFreeBuyingPowerPercent => base.RequiredFreeBuyingPowerPercent;
+
+        public override void ApplySplit(List<OrderTicket> tickets, Split split)
+        {
+            base.ApplySplit(tickets, split);
+        }
+
+        public override bool CanExecuteOrder(Security security, Order order)
+        {
+            return base.CanExecuteOrder(security, order);
+        }
+
+        public override bool CanSubmitOrder(Security security, Order order, out BrokerageMessageEvent message)
+        {
+            return base.CanSubmitOrder(security, order, out message);
```

Comment:
We could add some order validation here (e.g. supported order types, security types, etc.)?

There are many examples in the other existing brokerage models

Reply:
Done!